examples/ex11p.cpp

```diff
@@ -261,7 +261,11 @@ int main(int argc, char *argv[])
          strumpack->SetPrintSolveStatistics(false);
          strumpack->SetKrylovSolver(strumpack::KrylovSolver::DIRECT);
          strumpack->SetReorderingStrategy(strumpack::ReorderingStrategy::METIS);
+#if STRUMPACK_VERSION_MAJOR >= 3
```

Comment:
I think it will be better not to have this in the example code. Can we have
```c++
strumpack->SetMC64Job(0);
```
or maybe skip this call altogether if `NONE` is the default value?

Reply:
MC64Job is en enum class, so the 0 should be casted. 
Would you want to change
``` STRUMPACKSolver::SetMC64Job( strumpack::MC64Job job ) ```
to 
``` STRUMPACKSolver::SetMC64Job( int job ) ```
? Then do a switch/case in the implementation?

The default is
[strumpack::MC64Job::MAX_DIAGONAL_PRODUCT_SCALING](http://portal.nersc.gov/project/sparse/strumpack/master/namespacestrumpack.html#a7e2344627f0a9e9f910c1c131598cfd5)
which finds a permutation to maximize the product of the diagonal elements and then does a row and column scaling. This is for stability, mostly necessary when there are zeros on the diagonal of the matrix.
There are other possible values, including a parallel option we recently added that requires the Combinatorial BLAS library.

I could add routines
```
strumpack->DisableMatching(); // set to NONE
strumpack->EnableMatching(); // set to MAX_DIAGONAL_PRODUCT_SCALING
strumpack->EnableParallelMatching(); // set to COMBBLAS
```
?

Reply:
If that works from Strumpack perspective, I'd prefer adding the three routines you mentioned in the end, use `strumpack->DisableMatching()` in Example 11, and do the `#if STRUMPACK_VERSION_MAJOR >= 3` switch inside the implementations in `strumpack.cpp`.

Thanks !